nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/excel/ExcelUtils.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.excel;
+
+import org.apache.poi.ss.usermodel.Row;
+
+public class ExcelUtils {
```

Comment:
Unless there are other utility methods, it is probably cleaner to localize `hasCells` instead of creating this one-method class.

Reply:
This method is used in 3 different classes ExcelHeaderSchemaStrategy (line 79), ExcelRecordReader (line 113) and ExcelSchemaInference (line 64). I did not want to have to duplicate the code.

Reply:
@exceptionfactory What would you like me to do for this? Leave as is, declare the method static in one class and have the others use it or duplicate it for all 3 classes that use it?

Reply:
Thanks, it is fine for now, leaving it as is sounds good.